api/handlers_e2e_test.go

```diff
@@ -576,6 +589,7 @@ func TestAccountMaxResultsLimit(t *testing.T) {
 		errStatus  int
 		errAddress basics.Address
 	}{
+		{exclude: []string{}},
```

Comment:
Similarly, the maxResults parameter is set to 14 here too.
```suggestion
		{exclude: []string{}, errStatus: http.StatusBadRequest},
```